Scripts/developer_scripts/create_cgal_test

```diff
@@ -113,7 +129,7 @@ compile_and_run()
       rm -f \$OUTPUTFILE
       COMMAND="./\$1"
       if [ -f \$1.cmd ] ; then
-        COMMAND="\$COMMAND \`cat \$1.cmd\`"
+        COMMAND="\$COMMAND \`envsubst < \$1.cmd\`"
```

Comment:
@maxGimeno That is this part that parse the cmd and replace on the fly data dir. Maybe `envsubst` is not available?

Reply:
@maxGimeno What is that command?! Why not use standard shell, for the same effect?!

Can you tell us which StackOverflow page recommended it?

Reply:
I imagine you read https://stackoverflow.com/a/17030953/1728537 and I would have preferred the bash version with `read`, `eval`, and `echo`, at https://stackoverflow.com/a/415698/1728537.

Reply:
The problem is that `envsubst` is not found in cygwin platforms.